docs/topics/logging.rst

```diff
@@ -193,6 +193,45 @@ to override some of the Scrapy settings regarding logging.
     Module `logging.handlers <https://docs.python.org/2/library/logging.handlers.html>`_
         Further documentation on available handlers
 
+Custom Log Formats
+-------------------
+
+Custom log format can be set for different actions by extending ``scrapy.logformatter.LogFormatter`` class.
+
+Each method of ``scrapy.logformatter.LogFormatter`` represents an action. All methods inherited from 
+``scrapy.logformatter.LogFormatter`` in your custom log formatting class must return a dictionary listing
+the parameters ``level``, ``msg`` and ``args`` which are going to be used for constructing the log message.
+Listed below is details of what each key represents :
+
+*   ``level`` is the log level for that action, you can use those from the python logging library:
+    :setting:`logging.DEBUG`, :setting:`logging.INFO`, :setting:`logging.WARNING`, :setting:`logging.ERROR`
```

Comment:
`:setting:` is a custom syntax we have in Scrapy for Scrapy settings. Here you should use regular code formatting, for example:

    ``logging.DEBUG``

Consider, however, linking the first mention of Python’s logging module to its documentation. There users can find more information about these constants.